druid/src/widget/label.rs

```diff
@@ -132,6 +134,16 @@ impl<T: Data> Label<T> {
         self
     }
 
+    /// Builder-style method for setting the font.
+    ///
+    /// The argument can be either a `&'static str` or a [`Key<&'static str>`].
+    ///
+    /// [`Key<&str>`]: struct.Key.html
```

Comment:
Does ```[`Key<&str>`]``` apply the link to ```[`Key<&'static str>`]```? I thought it needs to match. Also the link is incorrect and should be `../struct.Key.html`

Reply:
Oops, yeah I forgot to update that when I added the `'static`.